src/main/java/com/github/silaev/mongodb/replicaset/MongoDbReplicaSet.java

```diff
@@ -1279,7 +1293,7 @@ private String getReplicaSetReConfigUnsetSlaveDelay() {
             ).collect(Collectors.joining(
                 ";\n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.